MAINTAINERS.yml

```diff
@@ -2060,6 +2060,21 @@ USB:
   labels:
     - "area: USB"
 
+USB-C:
+  status: maintained
+  maintainers:
+    - sambhurst
+  collaborators:
+    - keith-zephyr
+  files:
+    - drivers/usb_c/
+    - dts/bindings/usb-c/
+    - include/zephyr/*/usb_c/
+    - include/zephyr/usb_c/
+    - samples/subsys/usb_c/
```

Comment:
Please add
```
    - subsys/usb/usb_c/

```

possibly also to CODEOWNERS file.

Reply:
done